source/MRViewer/MRRibbonMenu.cpp

```diff
@@ -1485,17 +1475,15 @@ bool RibbonMenu::drawTransformContextMenu_( const std::shared_ptr<Object>& selec
             item->second.item->action();
             ImGui::CloseCurrentPopup();
         }
-        if ( ImGui::IsItemHovered() )
-            ImGui::SetTooltip( "Transforms object and resets transform value to identity." );
+        ImGui::SetTooltipIfHovered( "Transforms object and resets transform value to identity.", menu_scaling() );
```

Comment:
mb store menu scaling once in the beginning of the function

Reply:
Fixed.